### PR TITLE
[README] Bugfix for image display

### PR DESCRIPTION
**Pull request by kparichay**
Add bugfix for readme where images were not being shown

**Self evaluation:**
1. Build test: [x]Passed [ ]Failed [ ]Skipped
2. Run test: [x]Passed [ ]Failed [ ]Skipped

Signed-off-by: Parichay Kapoor <pk.kapoor@samsung.com>

### CODE REVIEW

**Review by zhoonit — APPROVED** [review]
LGTM :)

File: Applications/KNN/README.md [hunk 1]
```diff
@@ -3,24 +3,24 @@
 Here is some toy example which is distinguish simple images.
 The Mobile ssd V2 tensor flow lite model is used for the feature extractor and Nearest Neighbor is used for the classifier. All the training and testing is done on the Galaxy S8.
 
-![image](https://github.com/nnstreamer/nntrainer/blob/master/docs/images/08b09a80-ef29-11e9-8303-475fd75f4b83.png)
+![image](https://github.com/nnstreamer/nntrainer/blob/main/docs/images/08b09a80-ef29-11e9-8303-475fd75f4b83.png?raw=true)
```
Comment (zhoonit) [inline Applications/KNN/README.md:6]: I wonder if we can do like `/docs/images/...`

Comment (zhoonit) [reply Applications/KNN/README.md:6]: Maybe @lhs8928 can help this out??

**Review by jijoongmoon — APPROVED** [review]
LGTM

File: Applications/KNN/README.md (hunk 1 above)
Comment (kparichay) [reply Applications/KNN/README.md:6]: yes, that can also be done.
@lhs8928 can update it maybe

**Pull request merged**